packages/main/src/Slider.js

```diff
@@ -0,0 +1,271 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import SliderTemplate from "./generated/templates/SliderTemplate.lit.js";
+import {
+	isDown,
+	isUp,
+	isTabNext,
+	isTabPrevious,
+} from "../../base/src/events/PseudoEvents.js";
+
+// Styles
+import SliderCss from "./generated/themes/Slider.css.js";
+
+/**
+ * @private
+ */
+const metadata = {
+	tag: "ui5-slider",
+	properties: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+		/**
+		 * Defines whether the <code>ui5-slider</code> is disabled
+		 * (default is set to <code>false</code>).
+		 * A disabled <code>ui5-slider</code> can't be pressed or
+		 * focused, and it is not in the tab chain.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		disabled: {
+			type: Boolean,
+		},
+
+		/**
+		 * Defines the currently selected value
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		value: {
+			type: String,
+		},
+
+		/**
+		 * Defines the label of the slider.
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		label: {
+			type: String,
+			defaultValue: "",
+		},
+
+		/**
+		 * Indicates if the slider is expanded.
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 */
+		_expanded: {
+			type: Boolean,
+		},
+
+		_items: {
+			type: Object,
+		},
+	},
+	slots: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+
+	},
+	events: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+		/**
+		 * Fires when the slider is expanded.
+		 */
+		expand: {},
+
+		/**
+		 * Fires when the slider is collapsed.
+		 */
+		collapse: {},
+
+		/**
+		 *  Fires when new value is selected.
+		 */
+		valueSelect: {
```

Comment:
I think for public API we need an event called "change" to be complaint with all input based components, perhaps instead of valueSelect. For the expand/collapse ones I am not sure.